wallet/src/Pos/Wallet/Aeson/ClientTypes.hs

```diff
@@ -53,15 +53,36 @@ instance ToJSON (CBackupPhrase mw) where
     toJSON =
         genericToJSON defaultOptions
 
+
+-- We use a custom parser to support
+-- {
+--   "bpToList": ["derive pulse priority impose execute balcony guard hidden
+--    snap unhappy alcohol next"]
+-- }
+-- as in addition to
+-- {
+--    "bpToList": ["derive", "pulse", "priority", "impose", "execute",
```

Comment:
@KtorZ @Anviking lately I have become paranoid in mentioning _any_ valid 12 word mnemonic in our source code.

If possible, I would be inclined to either not have them here or to have dummy ones (i.e. a selection of 12 random words not suitable for BIP39) 😉

Reply:
Yup :+1: Good one @adinapoli-iohk, let's stick to:

https://github.com/input-output-hk/cardano-sl/blob/develop/wallet/src/Pos/Util/Mnemonic.hs#L313-L324

When showing any example.